tests/native.js

```diff
@@ -234,3 +234,21 @@ Native["tests/recordstore/ReaderMIDlet.waitWriterWrote.()V"] = function() {
     }
   }));
 };
+
+Native["tests/background/DestroyMIDlet.sendDestroyMIDletEvent.(Ljava/lang/String;)V"] = function(midletClassName) {
```

Comment:
Nit: it seems unnecessary for this native to take a *midletClassName* parameter, as it's intended to be used only by *tests.background.DestroyMIDlet*.